Formula/b/bower-email.rb

```diff
@@ -0,0 +1,26 @@
+class BowerEmail < Formula
+  desc "Curses terminal client for the Notmuch email system"
+  homepage "https://github.com/wangp/bower"
+  url "https://github.com/wangp/bower/archive/refs/tags/1.1.1.tar.gz"
+  sha256 "4c041681332d355710aa2f2a935ea56fbb2ba8d614be81dee594c431a1d493d9"
+  license "GPL-3.0-only"
+
+  depends_on "gpgme" => :build
+  depends_on "mercury" => :build
+  depends_on "pandoc" => :build
+
+  depends_on "coreutils"
+  depends_on "ncurses"
+  depends_on "notmuch"
```

Comment:
```suggestion
  depends_on "notmuch"
  
  uses_from_macos "ncurses"
```
Depend on Homebrew's Ncurses only if the program doesn't build with system library